libs/feedback/src/lib/feedback-widget/feedback-widget.component.html

```diff
@@ -14,56 +14,47 @@ <h2 class="modal-title">Feedback</h2>
       <ng-container *ngIf="(messages$ | async) as message">
         <ng-container *ngIf="message.length">
```

Comment:
why do we have `messages$ | async` in like 5 places there?
This can be simplified